coral-component-accordion/src/tests/test.Accordion.js

```diff
@@ -132,6 +132,19 @@ describe('Accordion', function() {
         expect(item.label).not.to.be.null;
       });
     });
+
+    describe('#level', function() {
+      it('should set heading level for Accordion item headings', function(done) {
+        const el = helpers.build(window.__html__['Accordion.base.html']);
+        el.level = 4;
+        helpers.next(function() {
```

Comment:
Does it still work if you remove helpers.next (1 or both)?

Reply:
It does still pass without helpers. I've removed them, but an unrelated test is failing: https://github.com/adobe/coral-spectrum/pull/109/checks?check_run_id=895766057